src/main/java/com/exactpro/th2/common/schema/factory/AbstractCommonFactory.java

```diff
@@ -246,6 +254,71 @@ public GrpcRouter getGrpcRouter() {
         });
     }
 
+    /**
+     * Registers custom message router.
+     *
+     * Unlike the {@link #registerCustomMessageRouter(Class, MessageConverter, Set, Set)} the registered router won't have any additional pins attributes
+     * except {@link QueueAttribute#SUBSCRIBE} for subscribe methods and {@link QueueAttribute#PUBLISH} for send methods
+     *
+     * @see #registerCustomMessageRouter(Class, MessageConverter, Set, Set)
+     */
+    public <T> void registerCustomMessageRouter(
+            Class<T> messageClass,
+            MessageConverter<T> messageConverter
+    ) {
+        registerCustomMessageRouter(messageClass, messageConverter, Collections.emptySet(), Collections.emptySet());
+    }
+
+    /**
+     * Registers message router for custom type that is passed via {@code messageClass} parameter.<br>
+     *
+     * @param messageClass custom message class
+     * @param messageConverter converter that will used to convert message to bytes and vice versa
```

Comment:
@param messageConverter converter that will **be** used to convert message to bytes and vice versa